pylearn2/models/mlp.py

```diff
@@ -2719,6 +2726,30 @@ def get_monitoring_channels_from_state(self,
             rval["misclass"] = T.cast(incorrect, config.floatX).mean()
 
         return rval
+    
+    @wraps(Linear.cost)
```

Comment:
Here, it should wrap `ConvNonlinearity.cost`. Put only the `Notes` section in the docstring, so that it will automatically be appended to the one of the parent class.